### PR TITLE
Fix Earthcare Plot 

### LINKED ISSUE #270: Find latest LTP forecast
The latest EarthCARE LTP forecast should be found automatically

### LINKED ISSUE #271: Exclude nightly overpass from EarthCARE overview
Exclude nightly overpass from EarthCARE overview

### PR DESCRIPTION
**Pull request by JakobDeutloff**
Latest LTP forecast selected automatically and display of two overpasses during one day fixed. Coastlines removed since cartopy keeps crashing since I updated the environment with environment.yaml. No idea why. 

This closes #270 , closes #271

### CODE REVIEW

File: orcestra_book/earthcare.md [hunk 1]
```diff
@@ -128,7 +175,6 @@ def plot_tracks(tracks, dates, ax):
     ax.plot(lons, lats, label=f'{radius_km} km circle', color='k')
 
     # pimp plot
-    ax.coastlines()
```
Comment (lkluft) [inline orcestra_book/earthcare.md:131]: Adding coastlines works for me using our current environment

**Review by lkluft — CHANGES_REQUESTED** [review]
Thanks for fixing this! ☺️ 

I would really like to keep the coastlines, otherwise this seems ready to be merged 👍

**Comment by lkluft** [comment]
This fixes #270 , fixes #271

**Comment by JakobDeutloff** [comment]
Deleting the environment and creating it freshly did the trick

**Review by lkluft — APPROVED** [review]

**Pull request merged**